### PR TITLE
Fix repository card tags position issue on lower screen size

### LINKED ISSUE #690: Position of tags is broken on repository card
## I'm submitting a...

- [x] Bug report  
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [ ] Feature request
- [ ] Documentation issue or request

## Context
On the screen resolution around 1100px, the order of tags on the repository card is broken.

image

### PR DESCRIPTION
**Pull request by hrvojevu**
Per issue: https://github.com/ExtensionEngine/tailor/issues/690

### CODE REVIEW

**Comment by hrvojevu** [comment]
@markolovric same issue can still occur in screen sizes lower than 1263px, but at point question arises if we want to support such small screen sizes.

**Comment by bubafinder** [comment]
Correct! Depends on the minimum resolution we want to support. @underscope?

File: client/components/catalog/Card/Tags/index.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
         v-for="{ id, name, truncatedName } in tags"
         :key="id"
         @click:close="showDeleteConfirmation(id, name)"
-        color="blue-grey lighten-4"
+        color="blue-grey lighten-4 mb-1"
```
Comment (underscope) [inline client/components/catalog/Card/Tags/index.vue]: mb-1 on color 🤔?

File: client/components/catalog/Card/index.vue [hunk 2]
```diff
@@ -113,24 +113,27 @@ export default {
...
+    height: 17.25rem;
+  }
+
   &:hover {
     transform: translateY(-0.25rem);
   }
 }
 
 .card-body {
-  height: 11.5rem;
```
Comment (underscope) [inline client/components/catalog/Card/index.vue:126]: You can probably remove now overflow as well 🤔 ?

**Review by underscope — CHANGES_REQUESTED** [review]

**Comment by underscope** [comment]
@markolovric @hrvojevu  > 1200

**Pull request merged**